pkg/storage/gcp/gcp_test.go

```diff
@@ -39,7 +41,10 @@ func (s *Storage) clear() error {
 			return err
 		}
 	}
-	return nil
+	// since this is a unique bucket name (see getTestConfig), we need to
+	// not only delete all the objects in the bucket (above), we need to then
+	// delete the bucket itself
+	return s.bucket.Delete(ctx)
```

Comment:
Actually, I don't know if this will work...clean is going to be called multiple times by the compliance tests. You can put this functionality inside `testing.M` if you want, or you'd have to have to different "clean" functions that the compliance test must accept, one that will clean contents, and one that will do a "final clean up"